vitest.config.mts

```diff
@@ -0,0 +1,45 @@
+import { resolve, dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import { defineConfig } from 'vitest/config';
+
+const CURRENT_DIR = dirname(fileURLToPath(import.meta.url));
+const WORKSPACE_ROOT = resolve(CURRENT_DIR, './');
+
+// Checking the environment variables simplifies the scripts in the package.json
+// We use `cross-env BROWSER=true vitest` instead of `vitest --project "browser/*"`
+// Which allows us to run `pnpm test:browser --project "x-charts"` for example.
+const getWorkspaces = () => {
+  const getFill = () => {
+    const isJSDOM = process.env.JSDOM === 'true';
```

Comment:
Would we be able to avoid these env vars if we give each project a name and prefix with `jsdom-` or `browser-`? Then we can use the existing `--project=jsdom-*` flag to do the filtering?